### PR TITLE
Multiple paragraphs

### LINKED ISSUE #1624: Multiple Paragraphs in single Caption
**Describe the bug**
We're rendering multiple `<figcaption>` elements, when there should be multiple `<p>` elements within a single `<figcaption>`

**To Reproduce**

1. Go to https://www.test.bbc.co.uk/news/articles/cm55llxvqkgo
2. inspect first figure. See multiple figcaption elements, rather than multiple para

**Expected behavior**
Multiple paragraph elements are needed within a single `<figcaption>` per image. 

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1624

**Overall change:** Rendering multiple paragraphs in a single caption

**Code changes:**

- Rendering multiple paragraphs in a single caption

Screen Shot 2019-04-23 at 17 16 08

When running locally, you can look at storybook here: http://localhost:9001/?selectedKind=Figure&selectedStory=with%20a%20caption%20with%20multiple%20paragraphs%20with%20a%20link

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by twinlensreflex — DISMISSED** [review]
😎 looks great!

**Review by oluoluoxenfree — DISMISSED** [review]

**Review by Bopchy — DISMISSED** [review]

**Comment by sareh** [comment]
Tested this locally with Storybook example http://localhost:9001/?selectedKind=Figure&selectedStory=with%20a%20caption%20with%20multiple%20paragraphs%20with%20a%20link&full=1&addons=0&stories=0&panelRight=0 on an iOS device with voiceover. It read out one caption and multiple paragraphs of content.

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Comment by dr3** [comment]
Note: this doesnt fix https://github.com/bbc/simorgh/issues/1641 and there is still a error in the console

**Comment by sareh** [comment]
@dr3 Merge conflicts addressed and unique keys added, addressing issue #1641

**Review by Bopchy — APPROVED** [review]

**Review by dr3 — APPROVED** [review]
Not 100% confident about this, as [if paragraphBlocks returns null](https://github.com/bbc/simorgh/pull/1627/files#diff-e2618c01198d1b7e026145a9ddc9be03R34) we will try to [map over it](https://github.com/bbc/simorgh/pull/1627/files#diff-e2618c01198d1b7e026145a9ddc9be03R25) however this isn't the only place in the code we rely on expectations in the block structure. 

This can go in on the understanding it will be improved in future :)

**Comment by dr3** [comment]
Popped it in ready for test as there was commits after @jamesbrumpton's comment

**Comment by jamesbrumpton** [comment]
Still looks good. Happy for this to be merged.

File: src/app/containers/Caption/index.jsx [hunk 1]
```diff
@@ -10,36 +11,32 @@ import InlineLink from '../InlineLink';
 
 const componentsToRender = { fragment: Fragment, urlLink: InlineLink };
 
-const renderText = textBlocks => (
-  <Blocks blocks={textBlocks} componentsToRender={componentsToRender} />
+const renderParagraph = paragraphBlock => (
+  <Paragraph key={deepGet([0, 'model', 'text'], paragraphBlock)}>
```
Comment (pjlee11) [inline src/app/containers/Caption/index.jsx:15]: We aren't passing `script` here which is required in Paragraph, planning to address this in https://github.com/bbc/simorgh/pull/1692

**Pull request merged**